samples/dashboard/demos/climate/demo.js

```diff
@@ -309,10 +326,15 @@ async function setupDashboard() {
                 },
                 xAxis: {
                     type: 'datetime',
-                    visible: true,
+                    visible: false,
                     labels: {
                         format: '{value:%Y-%m-%d}'
                     }
```

Comment:
Why hide the X axis labels? Without them we don't know what we're looking at. I find that making them visible (by default) and using the default `labels.format` works well for this chart. A custom labels format is not recommended because it hardwires it regardless of what time span we're looking at.
<img width="459" alt="Skjermbilde 2023-01-04 kl  08 17 35" src="https://user-images.githubusercontent.com/227836/210503364-8981f120-9d6c-4243-a5a2-e37963f1a8a4.png">

Reply:
That was the initial design, it's going to be changed anyway so I'm going to leave it as it is for now :)